packages/client/hmi-client/src/components/models/tera-model-diagram.vue

```diff
@@ -325,7 +325,7 @@ const contextMenuItems = ref([
 // Render graph whenever a new model is fetched or whenever the HTML element
 //	that we render the graph to changes.
 watch(
-	[props.model, graphElement],
+	[() => props.model, graphElement],
```

Comment:
Are you sure it isn't this instead: `() => [props.model, graphElement]`

Reply:
Yeah, Vue will throw a warning for this if props.model is null, if you don't use the getter function syntax.